src/jdk.sctp/unix/classes/sun/nio/ch/sctp/SctpServerChannelImpl.java

```diff
@@ -247,12 +243,6 @@ public SctpChannel accept() throws IOException {
             InetSocketAddress isa = isaa[0];
             sc = new SctpChannelImpl(provider(), newfd);
```

Comment:
You don't need `sc` anymore - you can just return here.

Reply:
So changed in 5e8c8d7.